diff --git a/libdispatch/xcodeconfig/libfirehose_kernel.xcconfig b/libdispatch/xcodeconfig/libfirehose_kernel.xcconfig
index b213158..691c96f 100644
--- a/libdispatch/xcodeconfig/libfirehose_kernel.xcconfig
+++ b/libdispatch/xcodeconfig/libfirehose_kernel.xcconfig
@@ -27,7 +27,7 @@ OTHER_LDFLAGS =
 OTHER_CFLAGS = -mkernel -nostdinc -Wno-packed
 PUBLIC_HEADERS_FOLDER_PATH = /usr/include/os
 PRIVATE_HEADERS_FOLDER_PATH = /usr/local/include/kernel/os
-HEADER_SEARCH_PATHS = $(PROJECT_DIR) $(SDKROOT)/System/Library/Frameworks/Kernel.framework/PrivateHeaders $(SDKROOT)/System/Library/Frameworks/Kernel.framework/Headers $(SDKROOT)/usr/local/include/os $(SDKROOT)/usr/local/include/firehose
+HEADER_SEARCH_PATHS = $(PROJECT_DIR) $(FAKEROOT)/System/Library/Frameworks/Kernel.framework/PrivateHeaders $(SDKROOT)/System/Library/Frameworks/Kernel.framework/Headers $(FAKEROOT)/usr/local/include/os $(FAKEROOT)/usr/local/include/firehose
 STRIP_INSTALLED_PRODUCT = NO
 COPY_PHASE_STRIP = NO
 SEPARATE_STRIP = NO
